.github/workflows/pages.yml

```
@@ -0,0 +1,18 @@
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: asciidoctor-ghpages
+        uses: manoelcampos/asciidoctor-ghpages-action@v2.3.0
+        with:
+          asciidoctor_params: --attribute=nofooter
+          source_dir: docs/
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)